www/utils/cookie.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Parse a document.cookie for the given key
+ * @param {string} key - key to search for
+ * @return {string} - value of (if any) for given key in cookie
+ */
+function getCookie( key ) {
+  const pair = document.cookie.split('; ').filter( ( str ) => {
+    return key === str.split('=')[ 0 ];
+  });
+
+  if ( pair.length ) {
+    return pair[ 0 ].split('=')[ 1 ];
+  }
+}
+
+/**
+ * Set a cookie with the given key, value (expires in 1 day)
+ * @param {string} key - key for cookie
+ * @param {string} value - value of cookie
+ */
+function setCookie( key, value ) {
+  const oneDayMs = 1000 * 60 * 60 * 24;
```

Comment:
?

Reply:
One day in milliseconds:
1000 * 1 millisecond = 1 second
1 second * 60 = 1 minute
1 minute * 60 = 1 hour
24 * 1 hour = one day!